app/assets/javascripts/angular/controllers/mobile/messages.js

```diff
@@ -0,0 +1,197 @@
+(function() {
+  // Set CSRF Headers to be used for Submitting Messages via Ajax.
+  angular.module('wtsApp').config([
+    "$httpProvider", function($httpProvider) {
+      $httpProvider.defaults.headers.common['X-CSRF-Token'] = $('meta[name=csrf-token]').attr('content');
```

Comment:
This takes our CSRF-Token, generated by rubyrails, and grabs it so that when we make an $http.post request, we can pass it along and rubyrails will see it as a legitimate request.